dashboard/src/reducers/apps.ts

```diff
@@ -19,8 +19,14 @@ const appsReducer = (
       return { ...state, isFetching: true };
     case getType(actions.apps.receiveApps):
       return { ...state, isFetching: false, items: action.payload };
-    case getType(actions.apps.errorApps):
-      return { ...state, isFetching: false, error: action.payload };
+    case getType(actions.apps.errorGetApp):
+      return { ...state, isFetching: false, getError: action.payload };
```

Comment:
The same in the state - not sure why we can't just have one error which can be different custom types.